app/services/processors/blog_post_count_metrics_updater.rb

```diff
@@ -0,0 +1,53 @@
+module Processors
+  class BlogPostCountMetricsUpdater < BaseService
+    def call
+      Technology.all.each do |technology|
+        new_blog_posts_by_month_for(technology).each do |month, new_blog_post_count|
```

Comment:
`find_each` would be preferable maybe

Reply:
Where would you add it?
`new_blog_posts_by_month_for(technology)` returns a hash, not a relation, so we can't use it there.
And there should be very few records in `Technology` (not more than 10), so I don't think iterating though them in batches would do much.

Reply:
It would be something like this: `Technology.find_each do |technology|`, gets records in batches and not all at once. By default is a batch of 1000 records

Reply:
But if `Technology` won't have more than 10 records, 15 at most, would it make any difference?
(I'm not opposed to the change, I'm just trying to understand the benefit)

Reply:
I thought it was an Arel, in that case ignore my comment